2021-11-22 22:12:38,454 INFO    MainThread:24976 [internal.py:wandb_internal():91] W&B internal server running at pid: 24976, started at: 2021-11-22 22:12:38.453869
2021-11-22 22:12:38,457 DEBUG   HandlerThread:24976 [handler.py:handle_request():124] handle_request: check_version
2021-11-22 22:12:38,457 INFO    WriterThread:24976 [datastore.py:open_for_write():80] open: /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/run-37xpbixg.wandb
2021-11-22 22:12:38,461 DEBUG   SenderThread:24976 [sender.py:send():179] send: header
2021-11-22 22:12:38,462 DEBUG   SenderThread:24976 [sender.py:send_request():193] send_request: check_version
2021-11-22 22:12:38,539 DEBUG   SenderThread:24976 [sender.py:send():179] send: run
2021-11-22 22:12:38,786 WARNING MainThread:24976 [internal.py:wandb_internal():147] Internal process interrupt: 1
2021-11-22 22:12:38,815 INFO    SenderThread:24976 [dir_watcher.py:__init__():168] watching files in: /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/files
2021-11-22 22:12:38,815 INFO    SenderThread:24976 [sender.py:_start_run_threads():719] run started: 37xpbixg with start time 1637586757
2021-11-22 22:12:38,945 WARNING MainThread:24976 [internal.py:wandb_internal():147] Internal process interrupt: 2
2021-11-22 22:12:38,945 ERROR   MainThread:24976 [internal.py:wandb_internal():150] Internal process interrupted.
2021-11-22 22:12:39,540 INFO    WriterThread:24976 [datastore.py:close():288] close: /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/run-37xpbixg.wandb
2021-11-22 22:12:39,540 INFO    HandlerThread:24976 [handler.py:finish():658] shutting down handler
2021-11-22 22:12:39,815 INFO    SenderThread:24976 [sender.py:finish():947] shutting down sender
2021-11-22 22:12:39,816 INFO    SenderThread:24976 [dir_watcher.py:finish():282] shutting down directory watcher
2021-11-22 22:12:40,816 INFO    SenderThread:24976 [dir_watcher.py:finish():312] scan: /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/files
2021-11-22 22:12:40,816 INFO    SenderThread:24976 [dir_watcher.py:finish():318] scan save: /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/files/config.yaml config.yaml
2021-11-22 22:12:40,825 INFO    SenderThread:24976 [file_pusher.py:finish():177] shutting down file pusher
2021-11-22 22:12:40,826 INFO    SenderThread:24976 [file_pusher.py:join():182] waiting for file pusher
2021-11-22 22:12:41,704 INFO    Thread-10 :24976 [upload_job.py:push():137] Uploaded file /home/workspace/FedML/fedml_experiments/standalone/fedavg/wandb/run-20211122_221237-37xpbixg/files/config.yaml
2021-11-22 22:12:42,562 INFO    MainThread:24976 [internal.py:handle_exit():78] Internal process exited
